monitoring/monitor.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+echo --- General Information ---
+echo \#CPU: $(nproc)
+echo Total Memory: $(free -m | grep Mem | awk '{ print $2 }')  # prints in mb
+echo Total Disk space: $(df -k | grep cromwell_root | awk '{ print $2}')  # prints in kb
+echo
+echo --- Runtime Information ---
+
+function runtimeInfo() {
+        # echo [$(date)]  # we don't really care about the date that much.
+
+        # print memory usage as a percentage with up to 4 significant digits
+        echo \* Memory usage \(%\): $(free -m | grep Mem | awk '{ OFMT="%.2f"; print ($3/$2)*100; }')%
+        # print memory usage in megabytes
+        echo \* Memory usage \(MB\): $(free -m | grep Mem | awk '{ print $3 }')
+        # print disk usage as a percentage with up to 4 significant digits
+        echo \* Disk usage \(%\): $(df -k | grep cromwell_root | awk '{ OFMT="%.2f"; print ($3/$2)*100; }')%
+        # print disk usage in 1024 k blocks
+        echo \* Disk usage \(KB\): $(df -k | grep cromwell_root | awk '{ print $3; }')
+
+}
+
+while true; do runtimeInfo; sleep 5; done
```

Comment:
checking usage in each 5s. How long does one 10X submission last normally?

Reply:
It looks great. 'free' probably good enough for this purpose. alternatively, sometimes when I ran jobs on SGE, I used something like 'top -u username|grep command_name' to monitor jobs. it will automatically update and I pipe the output into a log file.

Reply:
Top was eating tons of CPU so I explicitly removed it from this monitoring script. I think top makes sense, but only if the interval between measurements is long.

Reply:
Would it be possible to run `parse_monitoring_log.py` on the `monitoring.log` here after this finishes?

Reply:
That's a great thought! 

Unfortunately, I think this script probably gets killed when the parent process completes, so there would be no guarantee that the script would run.

Reply:
Ok that makes sense!